packages/web-api-scan-request-sender/src/sender/scan-request-sender.ts

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { Queue, StorageConfig } from 'azure-services';
+import { inject, injectable } from 'inversify';
+import * as _ from 'lodash';
+import { OnDemandPageScanRunResultProvider, PageScanRequestProvider } from 'service-library';
+import { OnDemandPageScanRequest, OnDemandPageScanResult, OnDemandPageScanRunState, OnDemandScanRequestMessage } from 'storage-documents';
+
+@injectable()
+export class ScanRequestSender {
+    constructor(
+        @inject(PageScanRequestProvider) private readonly pageScanRequestProvider: PageScanRequestProvider,
+        @inject(OnDemandPageScanRunResultProvider) private readonly onDemandPageScanRunResultProvider: OnDemandPageScanRunResultProvider,
+        @inject(Queue) private readonly queue: Queue,
+        @inject(StorageConfig) private readonly storageConfig: StorageConfig,
+    ) {}
+
+    public async sendRequestToScan(onDemandPageScanRequests: OnDemandPageScanRequest[]): Promise<void> {
+        const docsList = onDemandPageScanRequests.map(onDemandScanReq => {
+            return onDemandScanReq.id;
+        });
+        console.log(
```

Comment:
Is this for debug only? If yes we need to remove it, otherwise use logger and expand message. .toJSON() is a string